frontend/src/lib/components/apps/editor/inlineScriptsPanel/InlineScriptsPanel.svelte

```diff
@@ -39,9 +39,13 @@
 			? findGridItem($app, $selectedComponentInEditor?.split('_')?.[0])
 			: undefined
 
-	$: hiddenInlineScript = $app?.hiddenInlineScripts?.findIndex((k_, index) =>
-		$selectedComponentInEditor?.startsWith(BG_PREFIX + index)
```

Comment:
`startsWith`will be true when $selectedComponentInEditor is bg_10 and index is 1